components/WarningBox.qml

```diff
@@ -44,19 +44,22 @@ Rectangle {
             font.family: MoneroComponents.Style.fontRegular.name
             font.pixelSize: root.fontSize
             horizontalAlignment: TextInput.AlignLeft
-            selectByMouse: true
             textFormat: Text.RichText
             wrapMode: Text.WordWrap
-            textMargin: 0
             leftPadding: 4
             rightPadding: 18
             topPadding: 10
             bottomPadding: 10
-            readOnly: true
             onLinkActivated: root.linkActivated();
 
             selectionColor: MoneroComponents.Style.textSelectionColor
             selectedTextColor: MoneroComponents.Style.textSelectedColor
```

Comment:
can drop these lines too

Reply:
And maybe replace `TextArea` with `Text`

Reply:
updated